diff --git a/after/syntax/c.vim b/after/syntax/c.vim
index aa4a31e..a25add1 100644
--- a/after/syntax/c.vim
+++ b/after/syntax/c.vim
@@ -25,6 +25,14 @@ syn match    cCustomFunc     "\w\+\s*(\@=" contains=cCustomParen
 hi def link cCustomFunc  Function
 
 " -----------------------------------------------------------------------------
+"  Highlight member variable names.
+" -----------------------------------------------------------------------------
+"syn match   cCustomDot    "\." contained
+"syn match   cCustomPtr    "->" contained
+"syn match   cCustomMemVar "\(\.\|->\)\w\+" contains=cCustomDot,cCustomPtr
+"hi def link cCustomMemVar Function
+
+" -----------------------------------------------------------------------------
 "  Source: aftersyntaxc.vim
 " -----------------------------------------------------------------------------
 
@@ -264,13 +272,13 @@ hi def link cBoolean Boolean
 " -----------------------------------------------------------------------------
 
 " Operators
-"syn match cOperator	"\(<<\|>>\|[-+*/%&^|<>!=]\)="
-"syn match cOperator	"<<\|>>\|&&\|||\|++\|--\|->"
-"syn match cOperator	"[.!~*&%<>^|=,+-]"
-"syn match cOperator	"/[^/*=]"me=e-1
-"syn match cOperator	"/$"
-"syn match cOperator "&&\|||"
-"syn match cOperator	"[][]"
+syn match cOperator	"\(<<\|>>\|[-+*/%&^|<>!=]\)="
+syn match cOperator	"<<\|>>\|&&\|||\|++\|--\|->"
+syn match cOperator	"[.!~*&%<>^|=,+-]"
+syn match cOperator	"/[^/*=]"me=e-1
+syn match cOperator	"/$"
+syn match cOperator "&&\|||"
+syn match cOperator	"[][]"
 "
 "" Preprocs
 "syn keyword cDefined defined contained containedin=cDefine
